wagtailtinymce/rich_text.py

```diff
@@ -56,7 +141,7 @@ def getDefaultArgs(cls):
             'options': {
                 'browser_spellcheck': True,
                 'noneditable_leave_contenteditable': True,
-                'language': translation.to_locale(translation.get_language()),
+                'language': translation.to_locale(translation.get_language() or 'en'),
```

Comment:
Hi, @nfletton

the django localization is not Compatible to the tinymce localization,
for example in django shell,
>>> from django.utils import translation
>>> translation.check_for_language('en')
True
>>> translation.check_for_language('en_US')
False

so, there are two solutions for it`s problem,
the first one,  fix it in initialization in TinyMCERichTextArea.
the second one, fix filename for js file name, but it ugly. 

i will fix it soon.